agentarchives/archivists_toolkit/client.py

```diff
@@ -1,5 +1,8 @@
 import logging
 
+import pymysql
+pymysql.install_as_MySQLdb()
+
 import MySQLdb
```

Comment:
However it looks like it doesn't do anything more than register a fake `MySQLdb` module to point to `pymysql`, so changed this to just `import pymysql as MySQLdb`.